graspy/inference/latent_position_test.py

```diff
@@ -78,11 +78,11 @@ class LatentPositionTest(BaseInference):
         The observed difference between the embedded positions of the two input graphs
         after an alignment (the type of alignment depends on ``test_case``)
 
-    p_value_1_, p_value_2_ : float
+    p_value_1\_, p_value_2\_ : float
         The p value estimated from the null distributions from sample 1 and sample 2.
 
     p_value_ : float
-        The overall p value from the test; this is the max of p_value_1_ and p_value_2_
```

Comment:
@bdpedigo @pauladkisson i wonder if it's these ones that are throwing that, bcuz they appear in text. otherwise null distribution would have the same error, me thinks. btw, shouldn't these should be in ticks or smth, similar to #429, or are ticks only for attributes?

Reply:
Good catch. It is indeed only variables with a trailing underscore that appear in text.  That's why p_value_ doesn't throw an error -- because it isn't referenced in a text block.

Reply:
i do think @alyakin314 is probably right about backticks, @pauladkisson have you seen examples of this?

Reply:
Backticks fixed the whole issue lol.  Will search the docs for similar errors with the back ticks.

Reply:
Hey @bdpedigo, the amount that this formatting spec is missing from the docs is extensive.  I think it may be cleaner if I raise a new issue and tackle it in a separate branch.  Is that alright?

Reply:
@pauladkisson I'm not sure what you mean by "this formatting spec" (all of the backtick issues throughout the package?) but I trust your judgement and am almost always happy to have multiple separate (smaller) PRs where it makes sense to do so.

Reply:
Yeah I mean having variable names enclosed in back ticks.